src/views/_proxied-dot-io/product-docs-landing/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactElement } from 'react'
+import CardLink from 'components/card-link'
+import FeaturedCard from './partials/featured-card'
+import UseCaseCard from './partials/use-case-card'
+import s from './style.module.css'
+import VAULT_CONTENT from './vault-content'
+
+function ProductDocsLanding(): ReactElement {
+  // Note: later we could use this view with products other than Vault,
+  // and put this content somewhere more author-friendly (eg DatoCMS).
+  const productSlug = 'vault'
```

Comment:
Very "hard-coded" for now, but with an eye on the possibility that this view will be re-used by other products, and with other content. (Note: we'd probably move to a "block"-based approach rather than have hard-coded "Use Cases" and "Developer" sections as well).

Reply:
Just an observation that this is very vault focused, and maybe it's not worth trying to posture it as re-usable until that time comes? We're going to have to make a bunch of changes anyway, so it might be clearer to call it `VaultDocsLanding` for now....not a huge deal though.

Reply:
I'd probably not opt for making a view yet even, and just putting this markup in `src/pages/_proxied-dot-io/vault/docs/index.tsx`. Not a very strong opinion though. 👍🏻

Reply:
Great feedback here - have renamed to `vault-docs-landing` 👍 Could try to remove the view entirely, but I think then we'd have to put `components` somewhere NOT in the `pages` folder (so as to not have each component render as a page) 🤔